Simulator/Utils/Global.cpp

```diff
@@ -70,10 +70,10 @@ int g_deviceId = 0;
 #endif // USE_GPU
 
 // A random number generator.
-MTRand rng(1);
+MTRand rng;
```

Comment:
Why is this an object, and `rgNormrnd` is a pointer? Should these be global? Perhaps answering that second question can be deferred to a later time by creating an issue for someone else to address.

Reply:
`rgNormrnd` needs to be created dynamically via the factory. If I kept it as an object, I would be bound to either `Norm` or `MTRand`. `rng` is an object because it's slightly faster and I can't think of a reason for it to be a pointer.

Reply:
But isn't it also created by the factory?

Reply:
Nope, `rng` (used for initialization) is always an `MTRand` instance. It's initialization seed was hardcoded in earlier. I could create it using the factory if you'd like though.

Reply:
Oh, OK, then this is fine. No need to use a factory.